### ISSUE TITLE
test `transactionObj` with populated transaction data & cleanup code

### ISSUE THREAD

**Issue by emaG3m** [nearprotocol/near-web3-provider#25]
Make sure `transactionObj` is working properly with populated transaction. Potentially DRY up code with `transactionReceiptObj` since they both output the similar objects

**Comment by emaG3m** [comment]
things to remember: 
- near `gas_price` coming up 0. por que?
- tx signature values `v, r, s` will those need to be 0 always?
- match obj order later (for diff readability's sake)

**Comment by bowenwang1996** [comment]
> por que?

You probably used `./scripts/start_unittest.py` which will set gas price to 0. You can use `./scripts/start_localnet.py --nodocker` instead.

**Comment by emaG3m** [comment]
> You probably used `./scripts/start_unittest.py` which will set gas price to 0. You can use `./scripts/start_localnet.py --nodocker` instead.

On branch `stable`, when I use that script, it doesn't seem to approve of the genesis file:
```
thread 'main' panicked at 'Failed to deserialize the genesis records.: Error("invalid type: integer `1`, expected a tuple of size 2", line: 254, column: 1)', /Users/emilywilliams/code/summa/near/nearcore/core/chain-configs/src/genesis_config.rs:211:9
```

**Comment by bowenwang1996** [comment]
You probably have old files. You can have a clean start by either removing `~/.near` or specifying `--home=<some_directory>`

**Comment by barbaraliau** [comment]
Resolved in #29
